srcpkgs/cmake-vala/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'cmake-vala'
+pkgname=cmake-vala
+version=1
+revision=1
+wrksrc="cmake-vala-r${version}"
+build_style=cmake
+depends="cmake"
+short_desc="Vala CMake modules"
+maintainer="John Eira <john.eira@web.de>"
+license="BSD-3-Clause"
+homepage="https://gitlab.com/vala-panel-project/cmake-vala"
+distfiles="https://gitlab.com/vala-panel-project/cmake-vala/-/archive/r${version}/cmake-vala-r${version}.tar.gz"
+checksum=f8801ce01ed7ced690229ef81c0809af212166d585a0de8cf2b276444fd9f942
+archs=noarch
```

Comment:
should be between build_style= and wkrsrc=

Reply:
ok